src/system/sys_profiler.cc

```diff
@@ -714,6 +714,23 @@ void Profiler::output(MPI_Comm comm) {
     }
 }
 
+void Profiler::output(MPI_Comm comm, string profiler_path) {
```

Comment:
Add the `profiler_path` parameter as optional to the existing methods.

Reply:
done